packages/statusbar/src/components/hover.tsx

```diff
@@ -162,7 +162,7 @@ export class Popup extends Widget {
       offset: {
         horizontal: aligned
       },
-      privilege: 'forceAbove',
+      privilege: 'below',
```

Comment:
Was this change intended? What is its purpose?

Reply:
@krassowski This is a left-over of an attempt of using a popup instead of a dialog. I will remove it.